src/ReachSets/ContinuousPost/TMJets/post.jl

```diff
@@ -0,0 +1,68 @@
+using TaylorModels: validated_integ
+
+function post(𝒫::TMJets,
+              𝑆::AbstractSystem, # {<:ImplicitContinuousSystem}
+              invariant::Union{LazySet, Nothing},
+              𝑂::Options)::ReachSolution{Zonotope}
+
+    # ==================================
+    # Initialization
+    # ==================================
+
+    𝑂 = TwoLayerOptions(merge(𝑂, 𝒫.options.specified), 𝒫.options.defaults)
+
+    # system of ODEs
+    f! = 𝑆.s
+    n = 𝑂[:n]
+
+    # initial and final times, and maximum allowed number of steps
+    t0 = 0.0
+    T = 𝑂[:T]
+    max_steps = 𝑂[:max_steps]
+
+    # unrap algorithm-specific options
+    abs_tol, orderQ, orderT = 𝑂[:abs_tol], 𝑂[:orderQ] 𝑂[:orderT]
+
+    # initial sets
+    X0 = convert(IntervalBox, 𝑆.x0)
+    q0 = mid(X0)
+    δq0 = sup.(X0) - mid(X0)
+
+    # returns a TaylorN vector, each entry corresponding to an indep variable
+    set_variables("x", numvars=length(q0), order=2*orderQ)
```

Comment:
Don't you need to assign this to some variable?

Reply:
[set_variables (see docs for examples)](http://www.juliadiff.org/TaylorSeries.jl/stable/userguide/#Many-variables-1) does return the variables "x". But it does more: it also modifies the *global* structure that holds the working variables and the order `_params_TaylorN_`.  The definition in question (and default value) is [here](https://github.com/JuliaDiff/TaylorSeries.jl/blob/master/src/parameters.jl#L31).

Reply:
Then if you do not need the result, can you update the comment please?

Reply:
👍  i changed the sentence to

> fix the working variables and maximum order in the global parameters struct (_params_TaylorN_)